digitaltwins/samples/service/typescript/invoke_component.ts

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+import { DigitalTwinServiceClient } from 'azure-iot-digitaltwins-service';
```

Comment:
I would suggest to use "invoke_component_command.ts" for a file name to be consistent with the JS sample.